solr/core/src/java/org/apache/solr/core/QuerySenderListener.java

```diff
@@ -102,4 +104,50 @@ public void close() {}
     }
     log.info("QuerySenderListener done.");
   }
+
+  private List<NamedList<Object>> queries2list(ArrayList<Object> queries) {
```

Comment:
Can we change this to convertQueriesToList or "convertToList" instead of the "2"?   Digits in method names is not a naming pattern that we've used in Solr ;-=)

Reply:
Sure, I've done that. Do you think there's any more needed to get this over the line? I'm thinking a ref guide update would be useful - happy to craft something!

Reply:
I've added a brief example to the ref guide - could enhance this if necessary!